<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="chaipoint-Inspiron-5370" name="TerminalIdNotInDb" tests="1" failures="1" timestamp="10 Jan 2020 12:26:21 GMT" time="0.101" errors="0">
  <testcase name="testErrorCode" time="0.096" classname="cool.fountain.online.web.test.errors.TestErrorsForPostReq">
    <failure type="java.lang.AssertionError" message="Status expected:&amp;lt;400 BAD_REQUEST&amp;gt; but was:&amp;lt;200 OK&amp;gt;

&amp;gt; POST https://testing.chaipoint.com/qa3/outlet-pos-api/api/outlet-master/getTerminalConfig
&amp;gt; WebTestClient-Request-Id: [1]
&amp;gt; Content-Length: [28]
&amp;gt; service.auth.key: [1234567]
&amp;gt; Content-Type: [application/json]

{
  &amp;quot;terminalId&amp;quot;: &amp;quot;999888&amp;quot;
}

&amp;lt; 200 OK OK
&amp;lt; Date: [Fri, 10 Jan 2020 12:26:21 GMT]
&amp;lt; Content-Type: [application/json;charset=UTF-8]
&amp;lt; Transfer-Encoding: [chunked]
&amp;lt; Connection: [keep-alive]
&amp;lt; Access-Control-Allow-Headers: [Content-Type, Cookie, service.auth.key]
&amp;lt; Access-Control-Allow-Credentials: [true]
&amp;lt; Access-Control-Max-Age: [86400]
&amp;lt; X-Content-Type-Options: [nosniff]
&amp;lt; X-XSS-Protection: [1; mode=block]
&amp;lt; Cache-Control: [no-cache, no-store, max-age=0, must-revalidate]
&amp;lt; Pragma: [no-cache]
&amp;lt; Expires: [0]
&amp;lt; X-Frame-Options: [DENY]

{&amp;quot;errorCode&amp;quot;:&amp;quot;INVALID_TERMINAL_ID&amp;quot;,&amp;quot;message&amp;quot;:&amp;quot;Terminal Id Doesn&amp;apos;t Exist&amp;quot;}
">
      <![CDATA[java.lang.AssertionError: Status expected:<400 BAD_REQUEST> but was:<200 OK>

> POST https://testing.chaipoint.com/qa3/outlet-pos-api/api/outlet-master/getTerminalConfig
> WebTestClient-Request-Id: [1]
> Content-Length: [28]
> service.auth.key: [1234567]
> Content-Type: [application/json]

{
  "terminalId": "999888"
}

< 200 OK OK
< Date: [Fri, 10 Jan 2020 12:26:21 GMT]
< Content-Type: [application/json;charset=UTF-8]
< Transfer-Encoding: [chunked]
< Connection: [keep-alive]
< Access-Control-Allow-Headers: [Content-Type, Cookie, service.auth.key]
< Access-Control-Allow-Credentials: [true]
< Access-Control-Max-Age: [86400]
< X-Content-Type-Options: [nosniff]
< X-XSS-Protection: [1; mode=block]
< Cache-Control: [no-cache, no-store, max-age=0, must-revalidate]
< Pragma: [no-cache]
< Expires: [0]
< X-Frame-Options: [DENY]

{"errorCode":"INVALID_TERMINAL_ID","message":"Terminal Id Doesn't Exist"}

	at org.springframework.test.web.reactive.server.ExchangeResult.assertWithDiagnostics(ExchangeResult.java:200)
	at org.springframework.test.web.reactive.server.StatusAssertions.assertStatusAndReturn(StatusAssertions.java:227)
	at org.springframework.test.web.reactive.server.StatusAssertions.isBadRequest(StatusAssertions.java:131)
	at cool.fountain.online.web.test.errors.TestErrorsForPostReq.testErrorCode(TestErrorsForPostReq.java:61)
	at org.springframework.test.context.testng.AbstractTestNGSpringContextTests.run(AbstractTestNGSpringContextTests.java:180)
Caused by: java.lang.AssertionError: Status expected:<400 BAD_REQUEST> but was:<200 OK>
	at org.springframework.test.util.AssertionErrors.fail(AssertionErrors.java:55)
	at org.springframework.test.util.AssertionErrors.assertEquals(AssertionErrors.java:82)
	at org.springframework.test.web.reactive.server.StatusAssertions.lambda$assertStatusAndReturn$4(StatusAssertions.java:227)
	at org.springframework.test.web.reactive.server.ExchangeResult.assertWithDiagnostics(ExchangeResult.java:197)
	... 34 more
... Removed 30 stack frames]]>
    </failure>
  </testcase> <!-- testErrorCode -->
</testsuite> <!-- TerminalIdNotInDb -->
